orderer/consensus/smartbft/util_network_test.go

```diff
@@ -119,7 +119,17 @@ func (ns *NetworkSetupInfo) AddNewNode() (map[uint64]*Node, *Node) {
 
 	ledgerToSyncWith := findLargestDifferentLedger(ns.nodeIdToNode, newNodeId)
 
+	clusterService := &cluster.ClusterService{
```

Comment:
why is this related to the PR?

Reply:
Otherwise [this test ](https://github.com/hyperledger/fabric/blob/main/orderer/consensus/smartbft/chain_test.go#L358) crashes “panic: runtime error: invalid memory address”.

https://github.com/hyperledger/fabric/actions/runs/14818289679/job/41602294024

Can't find clusterService